central/risk/manager/manager.go

```diff
@@ -229,6 +242,47 @@ func (e *managerImpl) CalculateRiskAndUpsertImage(image *storage.Image) error {
 	return nil
 }
 
+// skipImageUpsert will return true if an image should not be upserted into the store.
+func (e *managerImpl) skipImageUpsert(img *storage.Image) (bool, error) {
+	if features.ScannerV4Enabled.Enabled() && !scannedByScannerV4(img) && e.scannedByClairify(img) {
```

Comment:
is the `!scannedByScannerV4(img)` check needed? or is it a quick short-circuit?

Reply:
Thats right, it's not required and intended to be a short-circuit.